### ISSUE TITLE
Inflector helper doc specifications

### ISSUE THREAD

**Issue by dkhalife** [bcit-ci/CodeIgniter#4692]
The [doc ](http://www.codeigniter.com/user_guide/helpers/inflector_helper.html) should explicitly mention that this Helper is for English language only and there is no supported way of directly extending it to other languages.

Perhaps future plans might include this as part of the language Library instead in order to help profit from localization?

**Comment by jim-parry** [comment]
The current user guide *could* mention your suggestion, but it is not inconsistent with the other frameworks. Cake & Yii, for instance, have inflector helpers that are English only, without expressly stating that. 

As for future plans, this sounds like a can of worms :(
I did not find any frameworks that deal elegantly with multiple languages, hmmm.
The closest was https://github.com/ICanBoogie/Inflector, which handles a handful of languages.
In other words, something easier said than done!

**Comment by dkhalife** [comment]
Yup you are right, I suppose for multilingual cases one could simply add a few extra language lines, it's not a huge deal anyway, thanks!

**Comment by narfbg** [comment]
Nitpicking level: infinity.
